cmd/gdsutil/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/syndtr/goleveldb/leveldb"
```

Comment:
@rebeccabilbro one of the reasons there are so many changes in this file is because I reordered the peer management and gossip functions to keep everything together and moved the `decrypt` function to the bottom. I did this primarily to resolve the merge conflict when I merged v1.1 into this branch. However, this will make it tricky for you to work on gdsutil without this branch merged in. I'll wrap up the gossip implementation so we can merge it to unblock you ASAP.